corehq/apps/cloudcare/templates/cloudcare/formplayer_home.html

```diff
@@ -0,0 +1,137 @@
+        <script src="{% static 'formplayer/script/vendor/ba-tiny-pubsub.min.js' %}"></script>
+        <script src="{% static 'formplayer/script/vendor/shortcut.js' %}"></script>
+        <script src="{% static 'formplayer/script/webformsession.js' %}"></script>
+        <script src="{% static 'formplayer/script/formnav-all.js' %}"></script>
+        <script src="{% static 'formplayer/script/entrycontrols_full.js' %}"></script>
+        <script src="{% static 'formplayer/script/fullform-ui.js' %}"></script>
+        <script src="{% static 'formplayer/script/utils.js' %}"></script>
+    {% endcompress %}
+{% endblock %}
+
+{% block js-inline %} {{ block.super }}
+    <script>
+        var translatedStrings = {
+            saved: "{% trans "Form successfully saved" %}",
+            errSaving: "{% trans "Error saving!" %}",
+            errSavingDetail: "{% trans "Save failed. Details: " %}",
+            unknownError: "{% trans "Unknown error: " %}",
+            unknownErrorDetail: "{% trans "This can happen if you loaded CloudCare from a different address than the server address" %}",
+            saveAll: "{% trans "All changes saved!" %}",
+            newWindow: "{% trans "Open in New Window" %}",
+            deleteSaved: "{% trans "Permanently delete this form? You will not be able to return to it later." %}",
+            deleteSuccess: "{% trans "Form was deleted." %}",
+            deleteError: "{% trans "Error deleting form. Please try again and report an issue if this problem persists." %}",
+            caseListError: "{% trans "Unable to load the case list. If you are using a filter expression please double check the syntax and try again. Please report an issue if this problem persists." %}",
+            {% if request.couch_user.is_web_user %}
+                navigateDirty: "{% trans "You have not submitted this form. Click ‘Stay on this Page’ to continue working on this form. If you click 'Leave this Page' any data you have entered will not be saved." %}",
+                sidebarDirty: "{% trans "You have not submitted this form. Click ‘Cancel’ to continue working on this form. If you click 'OK' any data you have entered will not be saved." %}",
+            {% else %}
+                navigateDirty: "{% trans "You have not submitted this form. Click ‘Stay on this Page’ to continue working on this form. If you click 'Leave this Page' you can resume work on this form later." %}",
+                sidebarDirty: "{% trans "You have not submitted this form. Click ‘Cancel’ to continue working on this form. If you click 'OK' you can resume work on this form later." %}",
+            {% endif %}
+            synced: "{% trans "User Data Successfully Synced" %}",
+            errSyncing: "{% trans "Could not sync user data. Please try again and report a bug if issue persists." %}"
+        };
+        var apps = {{ apps|JSON }};
+        var language = "{{ language }}";
+        FormplayerFrontend.start(apps, language);
```

Comment:
doesn't this need to be a conditional that only `starts` if the flag is enabled? else uses touchforms?